packages/loaders/stories/2-Dots.stories.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Story, Meta } from '@storybook/react';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { Dots, IDotsProps } from '@zendeskgarden/react-loaders';
+
+export default {
+  title: 'Components/Loaders/Dots',
+  component: Dots
+} as Meta;
+
+const StyledExampleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+interface IDefaultStoryProps extends IDotsProps {
+  size: number;
+}
+
+export const Default: Story<IDefaultStoryProps> = ({ color, delayMS, duration, size }) => {
+  return (
+    <StyledExampleWrapper>
+      <Dots color={color} delayMS={delayMS} duration={duration} size={`${size}px`} />
+    </StyledExampleWrapper>
+  );
+};
+
+Default.argTypes = {
+  size: {
+    name: 'size (px)',
+    control: { type: 'range', min: 30, max: 250, step: 1 }
+  },
+  duration: {
+    name: 'duration (ms)',
+    control: { type: 'range', min: 625, max: 2500, step: 10 }
+  },
+  color: { control: 'color' }
+};
+
+Default.args = {
+  size: 48,
+  duration: 1250,
+  color: PALETTE.blue[500]
```

Comment:
Can `size` and `color` start out as `undefined` so we can observe the `inherited` default?

Reply:
Our default designs for `Dots` have the color at `blue-500`. Should we apply a color of `blue-500` to the example wrapper to ensure that it shows?

Reply:
Yep.

Reply:
Updated to inherit from a wrapping element.